<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBusinessActivity</class>
 <widget class="QDialog" name="FormBusinessActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manutenção em Ramo de Atividades...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</iconset>
  </property>
  <widget class="QGroupBox" name="GBoxDataBusinessActivity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>161</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="title">
    <string>Dados</string>
   </property>
   <widget class="QLabel" name="LabelCodeBusinessActivity">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelNameBusinessActivity">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramo da Empresa:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelShortNameBusinessActivity">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome Curto:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonSaveBusinessActivity">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Salvar os Dados para o Ramo de Atividade...</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Filesave-48x48.png</normaloff>:/images/PNG/48x48/Filesave-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="CBoxCodeBusinessActivity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar um Código para o Ramo de Atividade...</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditIDBusinessActivity">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditNameBusinessActivity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o uma Descrição para o Ramo de Atividade da Empresa...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditShortNameBusinessActivity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar o Nome/Sigla/Apelido para o Ramo de Atividade da Empresa...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GBoxBusinessActivityStatusActive">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QCheckBox" name="CheckBoxBusinessActivityStatusActive">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Permite Ativar o Ramo de Atividade para ser Usado...</string>
     </property>
     <property name="text">
      <string>Ativo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GBoxTableViewBusinessActivity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Ramos de Atividades:</string>
   </property>
   <widget class="QTableView" name="TableViewBusinessActivity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ramos de Atividades de Empresas Atuais...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonRemoveBusinessActivity">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remoção do Ramo de Atividade  Atual...</string>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/delete-48x48.png</normaloff>:/images/PNG/48x48/delete-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonCleanBusinessActivity">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Limpar os Campos de Cadastro da Janela...</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Rebuild-48x48.png</normaloff>:/images/PNG/48x48/Rebuild-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="TButtonExitBusinessActivity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sair da Janela de Manutenção de Ramo de Atividade...</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
